### PR TITLE
🧰 CRT Maintenance 

### PR DESCRIPTION
**Pull request by WRadoslaw**
#4937

### CODE REVIEW

File: packages/atlas/src/views/studio/CrtWelcomeView/CrtWelcomeView.tsx [hunk 1]
```diff
@@ -1,26 +1,28 @@
-import { CrtView } from '@/views/studio/CrtView'
+import { atlasConfig } from '@/config'
+import { CrtMaintenanceView } from '@/views/studio/CrtMaintenanceView'
 
 export const CrtWelcomeView = () => {
-  return (
-    <CrtView />
```
Comment (thesan) [inline packages/atlas/src/views/studio/CrtWelcomeView/CrtWelcomeView.tsx]: Why is this removed ?

Comment (WRadoslaw) [reply packages/atlas/src/views/studio/CrtWelcomeView/CrtWelcomeView.tsx]: Mb, PR should target CRT branch. That way we no longer need a coming soon view which is what `crtView` was. Users will either see a landing page to create token or dashboard
